### DISCUSSION TITLE
how to group concat some field

### DISCUSSION THREAD

**Discussion by wvq** [prisma/prisma#9026] [Q&A]
how to select with group concat :
| user_id | other_field | class_id |
| --- | --- | --- |
|  1 | Lily | 1 |
| 1 | Lily | 2 |

```sql
select user_id, other_field, group_concat(class_id)  from user_class group by user_id
```

how to user prisma to make this query

**Comment by ryands17** [comment]
@wvq 👋 

This is only possible using a [raw query](https://www.prisma.io/docs/concepts/components/prisma-client/raw-database-access). It would be great if you could create a feature request [here](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=) for this.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
